content/games.html

```diff
@@ -165,7 +188,13 @@
 <h3>Server Browser</h3>
 <noscript><div class="noscript">Please enable JavaScript to view the server list.</div></noscript>
 <div id="servercontainer" style="display: none">
-<p><span id="activePlayers">0</span> players are online in <span id="activeServers">0</span> advertised servers.</p>
+<p><span id="activePlayers">0</span> players are online in <span id="activeServers">0</span> advertised servers matching the filters below.</p>
```

Comment:
The wording here doesn't quite feel right.  How about moving this below the checkboxes, and then changing it to `<p>Found <span id="activeServers">0</span> servers with <span id="activePlayers">0</span> players matching your selection:</p>?`